pkg/controller/remotemachineset/openstackactuator.go

```diff
@@ -76,6 +76,7 @@ func (a *OpenStackActuator) GenerateMachineSets(cd *hivev1.ClusterDeployment, po
 	computePool := baseMachinePool(pool)
 	computePool.Platform.OpenStack = &installertypesosp.MachinePool{
 		FlavorName: pool.Spec.Platform.OpenStack.Flavor,
+		Zones:      []string{""},
```

Comment:
It looks like Hive needs to start supporting zones in MachinePools for OpenStack.
https://github.com/openshift/installer/blob/96579442a7526367e86cf916ec85089c18dfaf24/pkg/types/openstack/machinepool.go#L26-L29

Reply:
Agreed. Created a Jira card on our backlog.